templates/event/event_edit.html.twig

```diff
@@ -72,4 +72,5 @@
 {% block javascripts %}
     {{ parent() }}
     <script src="/js/add_event_script.js"></script>
+    <script src="/js/add_picture_fix_script.js"></script>
```

Comment:
@daria-sieroshtan 
Better put in once in base template than copy-paste zillion times